bridge/src/sqlite3-monitor-state-store.ts

```diff
@@ -0,0 +1,85 @@
+import { IMonitorStateStore } from "./interfaces/monitor-state-store";
+import { Database } from "sqlite3";
+import { TransactionLocation } from "./types/transaction-location";
+
+export class Sqlite3MonitorStateStore implements IMonitorStateStore {
+    private readonly _database: Database;
+    private closed: boolean;
+
+    private constructor(database: Database) {
+        this._database = database;
+        this.closed = false;
+    }
+
+    static async open(path: string): Promise<Sqlite3MonitorStateStore> {
+        const database = new Database(path);
+        await this.initialize(database);
+        return new Sqlite3MonitorStateStore(database);
+    }
+
+    private static async initialize(database: Database): Promise<void> {
+        const CREATE_TABLE_QUERY = `CREATE TABLE IF NOT EXISTS monitor_states (
+            network TEXT PRIMARY KEY,
+            block_hash TEXT NOT NULL,
+            tx_id TEXT NOT NULL
+        )`;
+        return new Promise((resolve, error) => {
+            database.run(CREATE_TABLE_QUERY, e => {
+                if (e) {
+                    error();
+                } else {
+                    resolve();
+                }
+            });
+        });
```

Comment:
How about using [util.promisify()](https://nodejs.org/dist/latest-v8.x/docs/api/util.html)?

Reply:
I applied it in e9bcab4.